src/OpenTK.GraphicsLibraryFramework/Interfaces/IGLFW.cs

```diff
@@ -856,72 +1154,38 @@ internal interface IGLFW
         /// An array of button states, or <c>null</c> if the joystick is not present or an error occurred.
         /// </returns>
         /// <remarks>
+        /// <para>
         /// The returned array is allocated and freed by GLFW. You should not free it yourself.
         /// It is valid until the specified joystick is disconnected or the library is terminated.
-        ///
+        /// </para>
+        /// <para>
         /// This function must only be called from the main thread.
-        ///
-        /// TODO: Possible errors include <see cref="ErrorCode.NotInitialized"/>, <see cref="ErrorCode.InvalidEnum"/> and <see cref="ErrorCode.PlatformError"/>.
+        /// </para>
+        /// <para>
+        /// Possible errors include <see cref="ErrorCode.NotInitialized"/>, <see cref="ErrorCode.InvalidEnum"/> and <see cref="ErrorCode.PlatformError"/>.
+        /// </para>
         /// </remarks>
         unsafe byte* GetJoystickButtons(int jid, out int count);
 
         /// <summary>
+        /// <para>
         /// This function returns the state of all hats of the specified joystick.
-        /// Each element in the array is one of the following values:
```

Comment:
Is there something against that table?

Reply:
Table isn't necessary because there's an enum for that now.